freqtrade/freqtradebot.py

```diff
@@ -324,11 +324,10 @@ def create_trade(self, pair: str) -> bool:
             logger.info(f"Buy signal found: about create a new trade with stake_amount: "
                         f"{stake_amount} ...")
 
-            bidstrat_check_depth_of_market = self.config.get('bid_strategy', {}).\
-                get('check_depth_of_market', {})
-            if (bidstrat_check_depth_of_market.get('enabled', False)) and\
-                    (bidstrat_check_depth_of_market.get('bids_to_ask_delta', 0) > 0):
-                if self._check_depth_of_market_buy(pair, bidstrat_check_depth_of_market):
+            bid_check_dom = self.config.get('bid_strategy', {}).get('check_depth_of_market', {})
+            if ((bid_check_dom.get('enabled', False)) and
+                    (bid_check_dom.get('bids_to_ask_delta', 0) > 0)):
+                if self._check_depth_of_market_buy(pair, bid_check_dom):
```

Comment:
👍 
I left this part in this PR as it was to simplify the diff-list, to make the PR observable.